app/Http/Controllers/Backend/WikiController.php

```diff
@@ -10,22 +10,32 @@ class WikiController extends Controller
 {
     private static $with = ['wikiDb', 'wikiQueryserviceNamespace', 'settings'];
 
-    public function getWikiForDomain(Request $request): \Illuminate\Http\Response
+    public function getWikiForDomain(Request $request): \Illuminate\Http\JsonResponse
```

Comment:
I assume this change is ok as we really expect JSON here https://github.com/wbstack/mediawiki/blob/b864439a68bf29f9d81a124dc83a019ef3e92673/dist-persist/wbstack/src/Info/GlobalSet.php#L128